composer.json

```diff
@@ -12,33 +12,42 @@
         {
             "name": "Shivoham",
             "homepage": "https://github.com/Shivoham"
+        },
+        {
+            "name": "PedroTroller",
+            "homepage": "https://github.com/PedroTroller"
         }
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.3",
-        "symfony/http-kernel": "~2.5",
-        "symfony/dependency-injection": "~2.5",
-        "symfony/http-foundation": "~2.5",
-        "symfony/config": "~2.5",
-        "symfony/form": "~2.5",
-        "symfony/validator": "~2.5",
-        "twig/twig": "~1.11",
+        "php": ">=5.4",
+        "symfony/http-kernel": "~2.8 || ~3.0",
+        "symfony/dependency-injection": "~2.8 || ~3.0",
+        "symfony/http-foundation": "~2.8 || ~3.0",
+        "symfony/config": "~2.8 || ~3.0",
+        "symfony/form": "~2.8 || ~3.0",
+        "symfony/validator": "~2.8 || ~3.0",
+        "twig/twig": "~1.12|~2.0",
         "fzaninotto/faker": "^1.5"
     },
     "require-dev": {
+        "bossa/phpspec2-expect": "~1.0",
+        "henrikbjorn/phpspec-code-coverage": "^2.1",
+        "pedrotroller/php-cs-custom-fixer": "~1.4.0",
         "phpspec/phpspec": "~2.5",
-        "bossa/phpspec2-expect": "~1.0"
+        "phpspec/prophecy": "~1.6"
     },
     "config": {
         "bin-dir": "bin"
     },
     "autoload": {
-        "psr-0": { "Knp\\DictionaryBundle": "src/" }
+        "psr-4": {
```

Comment:
Wouldn't it be better to have an autoload in this form? 
```js 
"Knp\\DictionaryBundle\\": "./"
```
To avoid having this depth of folder?